cibuildwheel/util.py

```diff
@@ -13,14 +13,20 @@
 from typing import Dict, List, NamedTuple, Optional, Set
 
 import certifi
+import toml
 
 from .environment import ParsedEnvironment
-from .typing import PathOrStr
+from .typing import PathOrStr, PlatformName
 
-if sys.version_info < (3, 8):
-    from typing_extensions import Literal
+if sys.version_info < (3, 9):
+    from importlib_resources import files
 else:
-    from typing import Literal
+    from importlib.resources import files
+
+
+resources_dir = files('cibuildwheel') / 'resources'
```

Comment:
It supports directories that are zip files (see https://github.com/scikit-hep/particle/releases/tag/v0.14.0 - that `pyz runs directly without installing, `./particle.pyz`), and namespace packages that are split across directories.

Though, however, I'm using it here because it's more elegant and the "right' thing to do. We are accessing a resource.

PS: There's a few tricks when running from a zip file, you can open the path or access the filename, but cannot use the file name to open the file without a little workaround which I didn't attempt to implement.